### PR TITLE
Modified description and release

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
 # GCN Schema
 
-This repository contains the schema for GCN Notices as distributed by [GCN Kafka](https://gcn.nasa.gov). Schema are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
+This repository contains the schemas rthe essential schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.
```
Comment (Vidushi-GitHub) [inline README.md:3]:
```suggestion
This repository contains the schemas essential schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.
```

File: README.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
 # GCN Schema
 
-This repository contains the schema for GCN Notices as distributed by [GCN Kafka](https://gcn.nasa.gov). Schema are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
+This repository contains the schemas essential schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.
```
Comment (Vidushi-GitHub) [inline README.md:3]:
```suggestion
This repository contains the schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.
```

File: README.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
 # GCN Schema
 
-This repository contains the schema for GCN Notices as distributed by [GCN Kafka](https://gcn.nasa.gov). Schema are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
+This repository contains the schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.
 
-The preferred alert format for GCN Kafka is JSON. We have designed a set of core schema which serve as the building blocks for new GCN Notices. Instrument-specific schema can also be created, but we request that you utilize the core schema as much as possible.
+The alert format for new Notices over GCN Kafka is JSON. We have designed a set of core schema which serve as the building blocks for new GCN Notices. Instrument-specific schema can also be created, but we request that you utilize the core schema as much as possible.
```
Comment (Vidushi-GitHub) [inline README.md:5]:
```suggestion
The alert format for new Notices over GCN Kafka is JavaScript Object Notation (JSON), a widely adopted internet data format. We have designed a set of core schema which serve as the building blocks for new GCN Notices. Instrument-specific schema can also be created, but we request that you utilize the core schema as much as possible.
```

File: README.md [hunk 4]
```diff
@@ -1,26 +1,26 @@
...
     This command will handle the intermediate steps of updating and committing the path changes in each file as defined in the `version` and `postversion` npm scripts
 
 3.  Review the changes with `git log -p` to make sure that each file is appropriately updated.
 
-4.  Push the commit and tag:
+4.  Finally, push both the commit and the tag:
 
         git push && git push origin <tag name>
+
+This will make release officially available for use.
```
Comment (lpsinger) [inline README.md:26]:
```suggestion
This will make release available for use.
```

File: README.md [hunk 5]
```diff
@@ -1,26 +1,26 @@
 # GCN Schema
 
-This repository contains the schema for GCN Notices as distributed by [GCN Kafka](https://gcn.nasa.gov). Schema are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
+This repository contains the schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). These schemas are one of the important step in the process of establishing [New Notice](https://gcn.nasa.gov/docs/producers) for both existing and new missions.
```
Comment (lpsinger) [inline README.md:3]: "These schema are one of the important step": grammatical error, number agreement. Also, this wording begs the question what the _other_ important steps are.

"process of establishing New Notice for both existing and new missions": "New" should not be capitalized. Also, it looks like there is a missing word here.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Vidushi-GitHub) [reply README.md:3]:
```suggestion
This repository contains the schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). The formulation of mission schemas are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
```

File: README.md [hunk 6]
```diff
@@ -1,26 +1,26 @@
 # GCN Schema
 
-This repository contains the schema for GCN Notices as distributed by [GCN Kafka](https://gcn.nasa.gov). Schema are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
+This repository contains the schemas necessary for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). The formulation of mission schemas are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
```
Comment (lpsinger) [inline README.md:3]: "Necessary" is redundant.

```suggestion
This repository contains the schemas for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). The formulation of mission schemas are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
```

File: README.md [hunk 7]
```diff
@@ -1,26 +1,26 @@
...
 
-The preferred alert format for GCN Kafka is JSON. We have designed a set of core schema which serve as the building blocks for new GCN Notices. Instrument-specific schema can also be created, but we request that you utilize the core schema as much as possible.
+The alert format for new Notices over GCN Kafka is JavaScript Object Notation (JSON), a widely adopted internet data format. We have designed a set of core schema which serve as the building blocks for new GCN Notices. Instrument-specific schema can also be created, but we request that you utilize the core schema as much as possible.
```
Comment (lpsinger) [inline README.md:5]:
```suggestion
The alert format for new Notices over GCN Kafka is JavaScript Object Notation (JSON), a widely adopted Internet data format. We have designed a set of core schema which serve as the building blocks for new GCN Notices. Instrument-specific schema can also be created, but we request that you utilize the core schema as much as possible.
```

File: README.md [hunk 8]
```diff
@@ -1,26 +1,26 @@
...
 
 ## How To Release
 
 1.  Clone the parent repo (nasa-gcn/gcn-schema).
 
         git clone git@github.com:nasa-gcn/gcn-schema.git
+        cd gcn-schema
 
-2.  Run `npm version [ major | minor | patch ]`
+2.  Use the following command to initiate the [release](https://semver.org) process: `npm version [ major | minor | patch ]`
```
Comment (lpsinger) [inline README.md:16]:
```suggestion
2.  Tag a new version by running the following command: `npm version [ major | minor | patch ]`. Choose `major`, `minor`, or `patch` depending on the kind of update according to [Semantic Versioning](https://semver.org) rules.
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Comment by Vidushi-GitHub** [comment]
Looks better, thanks!

File: README.md [hunk 9]
```diff
@@ -1,26 +1,26 @@
 # GCN Schema
 
-This repository contains the schema for GCN Notices as distributed by [GCN Kafka](https://gcn.nasa.gov). Schema are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
+This repository contains the schemas for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). The formulation of mission schemas are one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
```
Comment (lpsinger) [inline README.md:3]:
```suggestion
This repository contains the schemas for crafting new GCN notices. It also includes mission-specific schemas and illustrative examples, which are distributed by [GCN Kafka](https://gcn.nasa.gov). The formulation of mission schemas is one step in the process for setting up [New Notice Producers](https://gcn.nasa.gov/docs/producers).
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Review by lpsinger — APPROVED** [review]

**Pull request merged**